web/static/js/components/vote_counter.jsx

```diff
@@ -40,7 +43,10 @@ class VoteCounter extends React.Component {
             transitionEnterTimeout={250}
             transitionLeaveTimeout={250}
           >
-            <div key={voteCountForIdea}>{voteCountForIdea}</div>
+            {stage === "action-items"
```

Comment:
@tnewell5 Rather than repeating the `<div>` (and, in turn, the key and innerValue) with different variables, let's just have a single variable, `voteCountForIdea` that we interpolate _into_ the `key` and `innerValue`. The naming of the variable will be clear enough for anyone who reads the logic, as it's a single `if (stage === 'voting') {} else {}`